packages/opencensus-exporter-ocagent/package-lock.json

```diff
@@ -94,12 +96,14 @@
     "@types/mocha": {
       "version": "2.2.48",
       "resolved": "https://registry.npmjs.org/@types/mocha/-/mocha-2.2.48.tgz",
-      "integrity": "sha512-nlK/iyETgafGli8Zh9zJVCTicvU3iajSkRwOh3Hhiva598CMqNJ4NcVCGMTGKpGpTYj/9R8RLzS9NAykSSCqGw=="
+      "integrity": "sha512-nlK/iyETgafGli8Zh9zJVCTicvU3iajSkRwOh3Hhiva598CMqNJ4NcVCGMTGKpGpTYj/9R8RLzS9NAykSSCqGw==",
+      "dev": true
```

Comment:
Why did you add dev: true everywhere?

Reply:
package-lock.json is automatically generated. If ```"dev": true``` then this dependency is either a development dependency ONLY of the top level module or a transitive dependency of one. (Source: https://docs.npmjs.com/files/package-lock.json#dev)